application/routes.py

```diff
@@ -16,4 +16,14 @@ def index():
         return redirect(url_for('index'))
 
     to_dos = Task.query.all()
-    return render_template('index.html', to_dos=to_dos, form=form)
\ No newline at end of file
+    return render_template('index.html', to_dos=to_dos, form=form)
+
+
+@app.route('/delete/<int:task_id>')
+def delete(task_id):
+    task = Task.query.filter_by(id=task_id).first()
```

Comment:
can you use query.get instead of query.filter?

Reply:
sure, since i`m trying to get a record by it`s primary key in my case.
Eagle eye )